op-node/node/node.go

```diff
@@ -189,6 +213,30 @@ func (n *OpNode) initL2(ctx context.Context, cfg *Config, snapshotLog log.Logger
 		return fmt.Errorf("failed to create Engine client: %w", err)
 	}
 
+	// Validate the L2 Client Chain ID
```

Comment:
What do you think about putting methods on the `cfg.Rollup`?

- `checkL1(backend) err`
- `checkL2(backend) err`

Then the logic can live there instead of here

cc @trianglesphere @protolambda

Reply:
needs a context passed in, but that works & seems pretty nice